modules/python.nix

```
@@ -0,0 +1,149 @@
+{ pkgs, ... }:
+let
+  pip = pkgs.callPackage ../pkgs/pip { };
+
+  poetry = pkgs.callPackage ../pkgs/poetry { };
+
+  python = pkgs.python310Full;
+
+  stderred = pkgs.replitPackages.stderred;
+
+  prybar = pkgs.replitPackages.prybar-python310;
+
+  pypkgs = pkgs.python310Packages;
+
+  debugpy = pypkgs.debugpy;
+
+  dapPython = pkgs.replitPackages.dapPython;
+
+  python-lsp-server = pkgs.callPackage ../pkgs/python-lsp-server { };
+
+  replit-py = pkgs.callPackage ../pkgs/replit-py { };
+
+  python-ld-library-path = pkgs.lib.makeLibraryPath [
+    # Needed for pandas / numpy
+    pkgs.stdenv.cc.cc.lib
+    pkgs.zlib
+    # Needed for pygame
+    pkgs.glib
+    # Needed for matplotlib
+    pkgs.xorg.libX11
+  ];
+
+  python3-wrapper = pkgs.stdenv.mkDerivation {
```

```suggestion
  python3-wrapper = pkgs.stdenvNoCC.mkDerivation {
```
We don't need C compilers for this derivation.